src/ast/statements/catch-stmt.ts

```diff
@@ -4,19 +4,23 @@ import { BlockStatement } from './block-stmt';
 import { ArrayBindingPattern } from '../expressions/array-binding-pattern';
 import { ObjectBindingPattern } from '../expressions/object-binding-pattern';
 import { Identifier } from '../expressions/identifier-expr';
+import { ExpressionNode } from '../expressions';
 
 /**
  * CatchClause.
  */
 export interface CatchClause extends SyntaxNode {
   readonly catchKeyword: SyntaxToken<TokenSyntaxKind>;
   readonly catchParameter: ArrayBindingPattern | ObjectBindingPattern | Identifier | null;
+  /* error recovery */
+  readonly initializer: ExpressionNode | null;
```

Comment:
@aladdin-add *never* try to "walk" this CST node - 'initializer'. It only exist for recovery purpose.